### PR TITLE
Abnormal Security apps doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create new doc for Abnormal Security app.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Abnormal Security App - Create Dochub Page](https://sumologic.atlassian.net/browse/CONN-3378)

### CODE REVIEW

File: blog-service/2024-07-17-apps.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - apps
+  - app-catalog
+  - abnormal-security
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Abnormal Security app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Tenable source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source/) to gain actionable insights and visualizations and helps you to prioritize and mitigate security incidents efficiently.
```
Comment (jpipkin1) [inline blog-service/2024-07-17-apps.md:14]:
```suggestion
We're excited to announce the release of the new Abnormal Security app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Abnormal Security source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source/) to gain actionable insights and visualizations and helps you to prioritize and mitigate security incidents efficiently.
```

**Review by jpipkin1 — APPROVED** [review]
This looks very good, @JV0812. I had one correction. And I added the Abnormal Security app to the Product List article (see commit "[Add Abnormal Security app to product list article](https://github.com/SumoLogic/sumologic-documentation/pull/4312/commits/2aa4d94e186aa1a65d911894e0a879f94b329be3)").

**Comment by JV0812** [comment]
Thanks John. I don't know why I keep missing to add in the Product list. I will add this to my checklist, so that I don't miss it from next time.

File: blog-service/2024-07-17-apps.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - abnormal-security
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Abnormal Security app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Abnormal Security source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source/) to gain actionable insights and visualizations and helps you to prioritize and mitigate security incidents efficiently.
+
+Check out our technical documentation [here](/docs/integrations/saas-cloud/abnormal-security/) to learn how you can set up and use the Abnormal Security app for Sumo Logic.
```
Comment (kimsauce) [inline blog-service/2024-07-17-apps.md:16]:
```suggestion
We are excited to announce the release of the new Abnormal Security app for Sumo Logic. This app leverages the Sumo Logic [Cloud-to-Cloud Abnormal Security source](https://github.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source/) to provide actionable insights and visualizations, helping you prioritize and mitigate security incidents efficiently.

For more details on setting up and using the Abnormal Security app, refer to our [technical documentation](https://github.com/docs/integrations/saas-cloud/abnormal-security/).
```

**Review by kimsauce — APPROVED** [review]

File: docs/integrations/saas-cloud/abnormal-security.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/abnormal-security-logo.png')} alt="abnormal-security-logo" width="90" />
+
+The Sumo Logic app for Abnormal Security offers robust monitoring of email security threats. It includes dashboards for an overview of total threats, detailed email threat analysis, and case management by severity and time trends. Key features highlight the threat types, attack vectors, and geolocation of senders, thereby aiding security teams in effectively identifying and responding to incidents. This app helps security teams to effectively monitor, identify, and respond to email-based threats, ensuring robust email security management. It offers actionable insights and visualizations to prioritize and mitigate security incidents efficiently.
+
+## Log type
+
+This app uses Abnormal Security Source to collect [threat logs](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats__threatId_) and [case logs](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Cases/get_cases) from the Abnormal Security platform.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/abnormal-security.md:16]:
```suggestion
This app uses the Abnormal Security Source to collect [threat logs](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats__threatId_) and [case logs](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Cases/get_cases) from the Abnormal Security platform.
```

File: docs/integrations/saas-cloud/abnormal-security.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+| where source_type matches ("case_log")
+| timeslice 1d
+| count as frequency by _timeslice
+
+```
+
+## Set up collection
+
+To set up [Cloud-to-Cloud Integration Abnormal Security Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source/) for the Abnormal Security app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Abnormal Security Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Abnormal Security app is properly integrated and configured to collect and analyze your Abnormal Security data.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/abnormal-security.md:117]:
```suggestion
To set up the Cloud-to-Cloud Integration Abnormal Security Source for the Abnormal Security app, follow the instructions provided [here](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source/). These instructions will guide you through the process of creating a source using the Abnormal Security Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Abnormal Security app is properly integrated and configured to collect and analyze your Abnormal Security data.
```

**Review by ankitgoelcmu — APPROVED** [review]
LGTM

**Review by kimsauce — APPROVED** [review]

**Pull request merged**